CODEOWNERS

```diff
@@ -37,6 +37,9 @@
 #model-assessment ui
 /libs/model-assessment @romanlutz @xuke444
 
+#e2e tests
+/libs/e2e @vinuthakaranth @tongyu-microsoft @romanlutz
```

Comment:
can I be added as well to the e2e tests?

Reply:
Sure, consider yourself signed up for writing a lot of them 😝